crates/orchestrator/src/tests/jobs/snos_job/mod.rs

```diff
@@ -0,0 +1,84 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use cairo_vm::vm::runners::cairo_pie::CairoPie;
+use chrono::{SubsecRound, Utc};
+use rstest::*;
+use starknet_os::io::output::StarknetOsOutput;
+use uuid::Uuid;
+
+use crate::constants::{CAIRO_PIE_FILE_NAME, SNOS_OUTPUT_FILE_NAME};
+use crate::jobs::snos_job::SnosJob;
+use crate::jobs::types::{JobItem, JobStatus, JobType, JobVerificationStatus};
+use crate::jobs::Job;
+use crate::tests::common::default_job_item;
+use crate::tests::config::TestConfigBuilder;
+use crate::tests::jobs::ConfigType;
+
+#[rstest]
+#[tokio::test]
+async fn test_create_job() {
+    let services = TestConfigBuilder::new().build().await;
+
+    let job = SnosJob.create_job(services.config.clone(), String::from("0"), HashMap::new()).await;
+
+    assert!(job.is_ok());
+    let job = job.unwrap();
+
+    let job_type = job.job_type;
+    assert_eq!(job_type, JobType::SnosRun, "job_type should be SnosRun");
+    assert!(!(job.id.is_nil()), "id should not be nil");
+    assert_eq!(job.status, JobStatus::Created, "status should be Created");
+    assert_eq!(job.version, 0_i32, "version should be 0");
+    assert_eq!(job.external_id.unwrap_string().unwrap(), String::new(), "external_id should be empty string");
+}
+
+#[rstest]
+#[tokio::test]
+async fn test_verify_job(#[from(default_job_item)] mut job_item: JobItem) {
+    let services = TestConfigBuilder::new().build().await;
+    let job_status = SnosJob.verify_job(services.config.clone(), &mut job_item).await;
+
+    // Should always be [Verified] for the moment.
+    assert_eq!(job_status, Ok(JobVerificationStatus::Verified));
+}
+
+#[rstest]
+#[tokio::test]
+async fn test_process_job() -> color_eyre::Result<()> {
+    dotenvy::from_filename("../.env.test")?;
+    // TODO: Hide this IP... private env variable of the repo?
```

Comment:
Ya let's do this

Reply:
nice, do we want to ignore the test or throw an error instead?

Reply:
I'd say ignore? It can happen that we don't have a pathfinder node and I don't think it should cause a fail